sixtyfps_compiler/widgets/fluent/sixtyfps_widgets_impl.60

```diff
@@ -0,0 +1,165 @@
+/* LICENSE BEGIN
+    This file is part of the SixtyFPS Project -- https://sixtyfps.io
+    Copyright (c) 2021 Olivier Goffart <olivier.goffart@sixtyfps.io>
+    Copyright (c) 2021 Simon Hausmann <simon.hausmann@sixtyfps.io>
+
+    SPDX-License-Identifier: GPL-3.0-only
+    This file is also available under commercial licensing terms.
+    Please contact info@sixtyfps.io for more information.
+LICENSE END */
+
+
+export global Palette := {
+    property<color> themeDarker: #004578;
+    property<color> themeDark: #005a9e;
+    property<color> themeDarkAlt: #106ebe;
+    property<color> themePrimary: #0078d4;
+    property<color> themeSecondary: #2b88d8;
+    property<color> themeTertiary: #71afe5;
+    property<color> themeLight: #c7e0f4;
+    property<color> themeLighter: #deecf9;
+    property<color> themeLighterAlt: #eff6fc;
+    property<color> black: #000000;
+    property<color> blackTranslucent40: rgba(0,0,0,0.4);
+    property<color> neutralDark: #201f1e;
+    property<color> neutralPrimary: #323130;
+    property<color> neutralPrimaryAlt: #3b3a39;
+    property<color> neutralSecondary: #605e5c;
+    property<color> neutralSecondaryAlt: #8a8886;
+    property<color> neutralTertiary: #a19f9d;
+    property<color> neutralTertiaryAlt: #c8c6c4;
+    property<color> neutralQuaternary: #d2d0ce;
+    property<color> neutralQuaternaryAlt: #e1dfdd;
+    property<color> neutralLight: #edebe9;
+    property<color> neutralLighter: #f3f2f1;
+    property<color> neutralLighterAlt: #faf9f8;
+    property<color> accent: #0078d4;
+    property<color> white: #ffffff;
+    property<color> whiteTranslucent40: rgba(255,255,255,0.4);
+    property<color> yellowDark: #d29200;
+    property<color> yellow: #ffb900;
+    property<color> yellowLight: #fff100;
+    property<color> orange: #d83b01;
+    property<color> orangeLight: #ea4300;
+    property<color> orangeLighter: #ff8c00;
+    property<color> redDark: #a4262c;
+    property<color> red: #e81123;
+    property<color> magentaDark: #5c005c;
+    property<color> magenta: #b4009e;
+    property<color> magentaLight: #e3008c;
+    property<color> purpleDark: #32145a;
+    property<color> purple: #5c2d91;
+    property<color> purpleLight: #b4a0ff;
+    property<color> blueDark: #002050;
+    property<color> blueMid: #00188f;
+    property<color> blue: #0078d4;
+    property<color> blueLight: #00bcf2;
+    property<color> tealDark: #004b50;
+    property<color> teal: #008272;
+    property<color> tealLight: #00b294;
+    property<color> greenDark: #004b1c;
+    property<color> green: #107c10;
+    property<color> greenLight: #bad80a;
+}
+
+
+export global StyleMetrics := {
+    property<length> layout-spacing: 8px;
+    property<length> layout-padding: 8px;
+    property<length> text-cursor-width: 2px;
+    property<color> window-background: white; //FIXME: Palette.white  does not compile (cannot access globals from other globals #175)
+    property<color> default-text-color: #201f1e; // neutralDark
```

Comment:
I changed that from "black" to the neutralDark color. But this breaks the cases/text/default_color.60 tests as it expect a black default color.

Should i change the test to expect this color?

Reply:
Since that would break with other styles, another option would perhaps be to go back to black for now and we do a separate change later to make all tests use a specific style for example? (regardless of the environnemtn variable / user override)

Reply:
I made the test indenpendent of the style